make check failed about util-linux@2.39 on suse15_aarch64 or ubuntu18_aarch64

hello, I meet a problem:make check failed about util-linux@2.39 on suse15_aarch64 or ubuntu18_aarch64

### Steps to reproduce the issue

```console
command:
./configure --disable-libmount --disable-swapon--disable-cramfs --disable-utmpdump --disable-hardlink --disable-lsfd --disable-blkid 
make -j16 check

error:
  CC       libuuid/src/unparse.o
  CC       tests/helpers/test_mkfds.o
  CC       tests/helpers/test_enosys.o
  CCLD     test_buffer
  CCLD     test_blkdev
  CCLD     test_canonicalize
  CCLD     test_colors
  CCLD     test_fileeq
tests/helpers/test_enosys.c:68:18: error: '__NR_move_mount' undeclared here (not in a function); did you mean '__NR_mount'?
  { "move_mount", __NR_move_mount },
                  ^~~~~~~~~~~~~~~
                  __NR_mount
tests/helpers/test_enosys.c:69:17: error: '__NR_open_tree' undeclared here (not in a function); did you mean '__NR_openat'?
  { "open_tree", __NR_open_tree },
                 ^~~~~~~~~~~~~~
                 __NR_openat
tests/helpers/test_enosys.c:70:14: error: '__NR_fsopen' undeclared here (not in a function); did you mean '__NR_mq_open'?
  { "fsopen", __NR_fsopen },
              ^~~~~~~~~~~
              __NR_mq_open
make[3]: *** [Makefile:10055: tests/helpers/test_enosys.o] Error 1
make[3]: *** Waiting for unfinished jobs....
  CCLD     test_fileutils
make[3]: Leaving directory '/home/stage/root/spack-stage-util-linux-2.39-d34pvatrrz6cd5eibar4ojmcngn35nbn/spack-src'
make[2]: *** [Makefile:16021: check-am] Error 2
make[2]: Leaving directory '/home/stage/root/spack-stage-util-linux-2.39-d34pvatrrz6cd5eibar4ojmcngn35nbn/spack-src'
make[1]: *** [Makefile:15722: check-recursive] Error 1
make[1]: Leaving directory '/home/stage/root/spack-stage-util-linux-2.39-d34pvatrrz6cd5eibar4ojmcngn35nbn/spack-src'
make: *** [Makefile:16025: check] Error 2
```
Can you tell me how to fix this problem?

--- a/tests/functions.sh
+++ b/tests/functions.sh
@@ -1158,3 +1158,9 @@ function ts_is_virt {
 	done
 	return 1
 }
+
+function ts_check_enosys_syscalls {
+	ts_check_test_command "$TS_CMD_ENOSYS"
+	"$TS_CMD_ENOSYS" ${@/#/-s } true 2> /dev/null
+	[ $? -ne 0 ] && ts_skip "test_enosys does not work: $*"
+}

--- a/tests/ts/misc/enosys
+++ b/tests/ts/misc/enosys
@@ -20,11 +20,8 @@ TS_DESC="enosys"
 . "$TS_TOPDIR"/functions.sh
 ts_init "$*"
 
-ts_check_test_command "$TS_CMD_ENOSYS"
 ts_check_test_command "$TS_HELPER_ENOSYS"
-
-"$TS_CMD_ENOSYS" true 2> /dev/null
-[ "$?" -eq "$TS_EXIT_NOTSUPP" ] && ts_skip "enosys does not work"
+ts_check_enosys_syscalls fallocate fsopen execve
 
 ts_init_subtest basic
 

--- a/tests/ts/mount/fallback
+++ b/tests/ts/mount/fallback
@@ -6,15 +6,13 @@ TS_DESC="fstab-fallback"
 . "$TS_TOPDIR"/functions.sh
 ts_init "$*"
 
-ts_check_test_command "$TS_CMD_ENOSYS"
 ts_check_test_command "$TS_CMD_MOUNT"
 ts_check_test_command "$TS_CMD_UMOUNT"
 ts_check_test_command "$TS_CMD_FINDMNT"
 ts_check_test_command "$TS_CMD_LOSETUP"
+ts_check_enosys_syscalls open_tree fsopen mount_setattr
 
 ts_skip_nonroot
-"$TS_CMD_ENOSYS" true 2> /dev/null
-[ "$?" -eq "$TS_EXIT_NOTSUPP" ] && ts_skip "enosys does not work"
 
 test_mount_fallback() {
 	ts_init_subtest "$1"

